compiler/modules/types/listable.bal

```diff
@@ -0,0 +1,148 @@
+// ListableTypes are types in which each expressible subtype can be reasonably
+// represented by a list of values. counter-eg: `int` is not a ListableType since
+// uint32 can't be reasonably represented by listing all values.
+type ListableType float|string;
```

Comment:
is `ListableType` a good name for this ?

Reply:
I suggest `EnumerableType`